.evergreen/auth_oidc/setup.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
+. $SCRIPT_DIR/../handle-paths.sh
+pushd $SCRIPT_DIR
+
+# Get the secrets
+rm -f secrets-export.sh
+. ./setup-secrets.sh
```

Comment:
Doesn't this remove the secrets before we source them?

Reply:
This is meant to be a "clean slate" file.